src/Sylius/Bundle/AdminBundle/composer.json

```diff
@@ -35,7 +35,9 @@
         "sylius/ui-bundle": "^1.6",
         "symfony/framework-bundle": "^5.4",
         "twig/intl-extra": "^2.12",
-        "twig/twig": "^2.12"
+        "twig/twig": "^2.12",
+        "knplabs/knp-menu": "^3.1",
+        "knplabs/knp-menu-bundle": "^3.0"
```

Comment:
Alphabetical order R.I.P [*]

Reply:
Btw, why do we need it added?

Reply:
@Zales0123 To run integration tests (why they were needed you can read in comment related to [src/Sylius/Bundle/AdminBundle/Tests/MessageHandler/Admin/SendResetPasswordEmailHandlerTest.php](https://github.com/Sylius/Sylius/pull/14138/files/6cd26878ccd8887421cf94617390a896d2df781f#diff-ee0687c388c61d7aa9653ad8091b0840adefdb1653a44fdc10e6627b30fa6008). I needed `Kernel` with loaded `AdminBundle` without `ApiBundle`. `AdminBundle` was complaining about missing service from `knp-menu-bundle` so I've added it.